packages/shared-lib/src/peripheralDevice/peripheralDeviceAPI.ts

```diff
@@ -26,16 +26,41 @@ export interface PiecePlaybackStartedResult extends PiecePlaybackCallbackData {
 }
 export type PiecePlaybackStoppedResult = PiecePlaybackStartedResult
 
-export type PlayoutChangedResults = PlayoutChangedResult[]
+export type PlayoutChangedResults = {
+	rundownPlaylistId: RundownPlaylistId
+	changes: PlayoutChangedResult[]
+}
+export enum PlayoutChangedType {
+	partPlaybackStarted = 'partPlaybackStarted',
+	partPlaybackStopped = 'partPlaybackStopped',
+	piecePlaybackStarted = 'piecePlaybackStarted',
+	piecePlaybackStopped = 'piecePlaybackStopped',
```

Comment:
Can we avoid camel case in enum members?

Reply:
I would highly suggest that we use SNAKE_CASE

Reply:
Good point! SNAKE_CASE it is!